src/clients/index.ts

```diff
@@ -5,5 +5,5 @@ export {
 } from "./AcrossConfigStoreClient";
 export { HubPoolClient } from "./HubPoolClient";
 export { SpokePoolClient } from "./SpokePoolClient";
-export { UBAClient } from "./UBAClient";
+export * from "./UBAClient";
```

Comment:
FWIW I used the web interface to propose this change. I'd inteded to propose this as a PR to this branch, but github pushed it directly instead. Would be good to confirm OK @james-a-morris.

Reply:
Works for me!